src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,45 @@
+package racingcar.domain;
+
+import racingcar.utils.RandomValueGenerator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * 경주에 참여하는 자동차들의 정보를 담당
+ * Car를 바탕으로 경주에 참여하는 그룹을 만든다.
+ * 각 자동차들은 랜덤값을 받아 현재 위치를 List 형태로 저장하게 된다.
+ */
+public class Cars {
+    private static final int MIN_INPUT_CAR_NUMBER = 1;
+
+    private List<Car> cars;
+
+    public Cars(int inputCarCount) {
+        validate(inputCarCount);
+        createCars(inputCarCount);
+    }
+
+    public List<Car> getCars() {
+        return cars;
+    }
```

Comment:
이렇게 cars를 그대로 반환하게 될 경우 일급컬렉션을 사용하는 의미가 많이 퇴색됩니다. 다음과 같이 스트림으로 반환하는건 어떨까요?
```suggestion
    public Stream<Car> stream() {
        return cars.stream();
    }
```

Reply:
준일님 리뷰 받으면서 새로운 걸 많이 배워가는 것 같아요!! :laughing:
정말 많이 배우고있습니다 감사합니다.